src/features/auth/signup/SetBirthday/DatePicker/DatePickerSpinner.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react'
+import DatePicker from 'react-native-date-picker'
+import styled from 'styled-components/native'
+import { v4 as uuidv4 } from 'uuid'
+
+import { DateInput } from 'features/auth/signup/SetBirthday/atoms/DateInput/DateInput'
+import { DatePickerProps } from 'features/auth/signup/SetBirthday/DatePicker/types'
+import { InputError } from 'ui/components/inputs/InputError'
+import { Spacer } from 'ui/theme'
+
+export function DatePickerSpinner(props: DatePickerProps) {
+  const [date, setDate] = useState<Date>(props.defaultSelectedDate)
```

Comment:
regarding the useEffect, which checks the nullity of `date`, isn't this type also `Date | undefined`?

Reply:
Ici il ne sera jamais `undefined` en natif car par défaut, la lib va mettre la date du jour / ou une date que que je peux définir

Reply:
Dans ce cas, on peut simplifier le useEffect 😉

Reply:
Fait ! 👍